tools/generator/filters/rust.py

```diff
@@ -0,0 +1,77 @@
+from functools import partial
+
+from . import register_filter
+from .common import generic_comment
+
+
+@register_filter
+def rust_call(func) -> str:
```

Comment:
Doesn't seem rust specific, maybe put that in common? Also, can't you just do that in the templates directly?